src/news/2018-09-17-GSoC2018.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Google Summer of Code 2018
+filename: GSoC2018
+date: '2018-09-17'
+---
+
+From March to August 2018, the Floe project participated in
+[Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/organizations/6584170814373888/#projects),
+a global program funded by Google which focuses on bringing student developers into open source
+software development. Floe team members mentored five students in a variety of projects, with the
+aim of expanding the Open Education Resources (OER) community and building connections with budding
+open source software developers.
+
+The projects that were selected for GSoC this year are diverse in quite a few respects. From
+technical details like the programming language, framework and style to more obvious aspects like
+their methods of interaction and presentation. That said, they all endeavoured to improve
+accessibility in computing and aid learning. The students presented their work at a
+[Community Workshop](https://wiki.fluidproject.org/display/fluid/Community+workshops)
+and a [recording of their presentations](https://idrc.cachefly.net/wiki.fluidproject.org/videos/IDRC_CommunityWorkshop_GoogleSummerOfCode_StudentPresentations_2018-08-15.mp4)
+is available. What follows is a brief description of each project and a link to its author’s final report.
+
+#### [Import external data to MyL3](https://summerofcode.withgoogle.com/archive/2018/projects/5924252133883904/)
+
+#### by Jeevan Prakash
```

Comment:
Please put this text at the end of the heading above:
```
#### [Import external data to MyL3](https://summerofcode.withgoogle.com/archive/2018/projects/5924252133883904/) by Jeevan Prakash
```

And please do the same for all project titles in this article

Reply:
The first three project titles look great! This is still an issue for the last two ("Playtown!" and "Upgrading Achecker")

Reply:
fixed